cmd/install/assets/hypershift_operator.go

```diff
@@ -941,7 +941,7 @@ type HyperShiftOperatorClusterRole struct {
 	EnableCVOManagementClusterMetricsAccess bool
 }
 
-func (o HyperShiftOperatorClusterRole) Build() *rbacv1.ClusterRole {
+func (o HyperShiftOperatorClusterRole) Build(managedService string) *rbacv1.ClusterRole {
```

Comment:
This is not the pattern we've been using for the build functions. Generally any params are set as fields, the Build() has no args

Reply:
Ahh I see now what you mean.